### PR TITLE
[ACS-7287] update node for docker images

### PR DESCRIPTION
**Pull request by akashrathod28**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [x] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
node.js older version for docker

**What is the new behaviour?**
update node.js version for docker images

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**: https://alfresco.atlassian.net/browse/ACS-7287

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "alfresco-content-app",
-  "version": "4.5.0",
+  "version": "4.4.1",
```
Comment (MichalKinas) [inline package.json:3]: I don't think this should be changed

**Review by MichalKinas — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # 1. Generate licenses
 
-FROM node:16.13-alpine3.14 AS builder
+FROM node:18.16-alpine3.17 AS builder
```
Comment (AleksanderSklorz) [inline Dockerfile:3]: Could you please do the same change for alfresco-applications as I can see there is incorrect version too?

File: package-lock.json
Comment (AleksanderSklorz) [inline package-lock.json:1]: this change should not be here

**Comment by AleksanderSklorz** [comment]
PR's title seems have incorrect format: 
image

File: Dockerfile (hunk 2 above)
Comment (akashrathod28) [reply Dockerfile:3]: yes raised a pr for same

File: package.json
[the remark this answers is not in the document]
Comment (akashrathod28) [reply package.json:1]: updated

File: package-lock.json
Comment (akashrathod28) [reply package-lock.json:1]: updated

File: package.json (hunk 1 above)
Comment (akashrathod28) [reply package.json]: removed changes from package file

**Review by MichalKinas — APPROVED** [review]

**Review by AleksanderSklorz — APPROVED** [review]

**Pull request merged**